### DISCUSSION TITLE
Complete German Translation

### DISCUSSION THREAD

**Discussion by X1000debug** [aclap-dev/video-downloadhelper#983] [Language & Translation]
[de.txt](https://github.com/user-attachments/files/16141978/de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
